<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor de Menu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="menuLayout"/>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>400</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Menu Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="menuNameLe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>400</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteMenuBtn">
       <property name="text">
        <string>Excluir Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importMenuBtn">
       <property name="text">
        <string>Importar Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportMenuBtn">
       <property name="text">
        <string>Exportar Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createMenuBtn">
       <property name="text">
        <string>Criar Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Menu ( preview ):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>523</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>523</width>
        <height>445</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Abas</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabLayout"/>
     </widget>
     <widget class="QWidget" name="page2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Bot√µes</string>
      </attribute>
      <layout class="QVBoxLayout" name="buttonLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;span style=&quot;color:red&quot;&gt;&lt;b&gt;Ordene arrastando a aba!&lt;/b&gt;&lt;/span&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>createMenuBtn</tabstop>
  <tabstop>exportMenuBtn</tabstop>
  <tabstop>importMenuBtn</tabstop>
  <tabstop>deleteMenuBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
